kiegroup/kie-wb-common
# JBPM-7604 Stunner - Showing case management properties for cases only

Hide / Show Case Management Properties whether the diagram is under a case project and the `adhoc` property is set to **true**

To test it is necessary 2 conditions:
- the existence of **.caseproject** file on the root project filesystem on the .niogit, on the WB just create a case project should be enough because it creates this file.
- the property adhoc set to true or false on the forms

Since the adhoc condition can be changed at runtime, you can test setting to true/false and see the Case Management properties be shown/hide.

@romartin 
@LuboTerifaj
